docker-compose.yml

```diff
@@ -9,31 +9,28 @@ services:
 #    volumes: this persists the db code locally
 #      - ./db/data/:/var/lib/postgresql/data
     environment:
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: 1234  # To be moved to environment file
-      POSTGRES_DATABASE: clinical_trials_status
-    expose:
-      - 5432
-    ports:
-      - 5432:5432
-    networks:
```

Comment:
Oh, so I hadn't included this network in the scraper which is why it couldn't connect. Makes sense to remove it in a build as simple as this